build.zig

```diff
@@ -162,42 +202,42 @@ pub fn buildLibSokol(b: *Build, options: LibSokolOptions) !*CompileStep {
             lib.linkFramework("MetalKit");
             lib.linkFramework("Metal");
         }
-        if (lib.target.getOsTag() == .ios) {
+        if (lib.rootModuleTarget().os.tag == .ios) {
             lib.linkFramework("UIKit");
             lib.linkFramework("AVFoundation");
             if (.gl == _backend) {
                 lib.linkFramework("OpenGLES");
                 lib.linkFramework("GLKit");
             }
-        } else if (lib.target.getOsTag() == .macos) {
+        } else if (lib.rootModuleTarget().os.tag == .macos) {
             lib.linkFramework("Cocoa");
             lib.linkFramework("QuartzCore");
             if (.gl == _backend) {
                 lib.linkFramework("OpenGL");
             }
         }
-    } else if (lib.target.getAbi() == .android) {
+    } else if (lib.rootModuleTarget().abi == .android) {
```

Comment:
I think there's an `isAndroid()` method now, might be the better choice because this is the only branch which checks for 'abi' instead of 'os.tag'.